packages/patternfly-4/react-core/src/components/ApplicationLauncher/ApplicationLauncherGroup.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import { DropdownGroup, DropdownGroupProps } from '../Dropdown';
+
+export const ApplicationLauncherGroup: React.FunctionComponent<DropdownGroupProps> = ({
```

Comment:
DropdownGroup only supports a string for a label. I could see where an icon and string might be desirable in the Application Launcher groups.